### DISCUSSION TITLE
Different endcaps depending on the position in the row (left or right) ?

### DISCUSSION THREAD

**Discussion by rbarzic** [The-OpenROAD-Project/OpenROAD-flow-scripts#1367] [Q&A]
Hello,

I'm trying to port the ORFS to a TSMC process. In this process, the standard cell library provided by a 3rd party contains 2 endcaps:   SEH_CAPR8 and  SEH_CAPL8. The SEH_CAPR8 cell is supposed to be placed on the right of the row, and the SEH_CAPL8 on the left.
The tapcell command seems to support only one type of endcap.  Any idea how this can be solved with the current flow?

**Comment by maliberty** [comment]
@gadfort is this supported with you recent work?
